app/assets/stylesheets/admin/content.css.scss

```diff
@@ -0,0 +1,11 @@
+.content {
```

Comment:
I think support for .css.scss extensions is going away. I would recommend moving everything to just .scss.